.github/workflows/scorecard.yml

```
@@ -0,0 +1,45 @@
+name: Scorecard supply-chain security
+on:
+  branch_protection_rule:
+  schedule:
```

It would be handy if we can run this by hand as well.

```suggestion
  workflow_dispatch:
  schedule:
```